extmod/cyw43_config_common.h

```diff
@@ -0,0 +1,143 @@
+/*
```

Comment:
I'm not sure, but maybe this file should be `extmod/cyw43-include/cyw43_config_common.h` or `extmod/cyw43-driver-include/cyw43_config_common.h`?

There are other common include dirs with this naming scheme.

Reply:
The other way we could do this is to move all three of `extmod/network_cyw.*` and this file to a directory like `extmod/cyw43/`. What do you think?

Reply:
Let's leave it where it is for now.  Can move it later if needed (eg in conjunction with factoring this into SDIO/SPI parts).